docs/cli.md

```diff
@@ -218,6 +219,19 @@ Usage:
 $ s2i rebuild <image name> [<new-tag-name>]
 ```
 
+# s2i generate
+The `s2i generate` command produces a Dockerfile using an existing S2I base 
+image that can be used to produce an image by any application supporting the
+format.
+
+The *image name* argument is in the URL-like, for example 
+`docker://docker.io/centos/nodejs-10-centos7`.
+
+Usage:
+
+```
+$ s2i generate <image name> <output file>
+```
```

Comment:
Could you follow it up with an example, please:

```
s2i generate docker://docker.io/centos/nodejs-10-centos7 Dockerfile.gen
```

Reply:
As with my previous comment, we should drop the `docker://` scheme. I rarely see examples of this in the field.

Reply:
Done in https://github.com/openshift/source-to-image/pull/1023/commits/cb94360b477113940ab8efd47411f12cb791bc84

Reply:
#1024

Reply:
Solved in https://github.com/openshift/source-to-image/pull/1023/commits/fb0be78c7f00a88b37cea07d4fc8e9e6ce59f9e9 and https://github.com/openshift/source-to-image/pull/1023/commits/888560cf0d71289ad9f706773c40c8974847f339